src/components/Section.tsx

```diff
@@ -8,16 +9,17 @@ export type Props = {
 };
 
 export default function Section({ title, href, children, id }: Props) {
+const {t} = useTranslation()
   return (
     <section id={id} className="py-4 desktop-container sm:py-8">
       <div className="mb-2 flex items-center justify-between px-2">
-        <div className="text-lg font-bold text-white sm:text-2xl">{title}</div>
+        <div className="text-lg font-bold text-white sm:text-2xl">{t(title)}</div>
```

Comment:
I think it shouldn't be responsible for translating. Let's keep the logic outside of this component